tests/components/TextAreaTest.tsx

```diff
@@ -126,6 +126,7 @@ describe('TextArea component', () => {
     const textAreaRef = createRef<HTMLTextAreaElement>();
 
     // Render the TextArea component and pass the ref as forwardRef
+    // eslint-disable-next-line no-unused-vars
     const { container } = render(<TextArea ref={textAreaRef} />);
```

Comment:
This could just be (without the eslint-disable):

// Render the TextArea component and pass the ref as forwardRef
render(<TextArea ref={textAreaRef} />);